requirements.txt

```diff
@@ -0,0 +1 @@
+requests
```

Comment:
specify the particular version of requests library. this is the recommended practice and you have to follow it